### PR TITLE
Add documentation for settings of the monitoring module

### PR DESCRIPTION
**Pull request by dgoetz**
Feel free to tell me if somethings requires adjustments like a better explanation of `PT1H`.

### CODE REVIEW

File: modules/monitoring/doc/03-Configuration.md [hunk 1]
```diff
@@ -17,6 +17,31 @@ Navigate into `Configuration` -> `Modules` -> `Monitoring`. This allows
...
+### Default Settings <a id="monitoring-module-configuration-settings"></a>
+
+Option                        | Description
+------------------------------|-----------------------------------------------
+acknowledge_expire            | **Optional.** Check "Use Expire Time" in Acknowledgement dialog by default. Defaults to **0 (false)**.
+acknowledge_expire_time       | **Optional.** Set default value for "Expire Time" in Acknowledgement dialog, its calculated as now + this setting. Format is a [PHP Dateinterval](http://www.php.net/manual/en/dateinterval.construct.php). Defaults to **1 hour (PT1H)**.
+acknowledge_notify            | **Optional.** Check "Send Notification" in Acknowledgement dialog by default. Defaults to **1 (true)**.
+acknowledge_persistent        | **Optional.** Check "Persistent Comment" in Acknowledgement dialog by default. Defaults to **0 (false)**.
+acknowledge_sticky            | **Optional.** Check "Sticky Acknowledgement" in Acknowledgement dialog by default. Defaults to **0 (false)**.
+custom_notification_broadcast | **Optional.** Check "Broadcast" in Custom Notification dialog by default. Only available with Backend "Icinga1". Defaults to **0 (false)**.
```
Comment (nilmerg) [inline modules/monitoring/doc/03-Configuration.md]: I'd say this should be removed. It's only relevant to Icinga 1.x environments which are not supported anymore. So we shouldn't cover such with new documentation as it's probably going to be removed entirely in the future.

Comment (dgoetz) [reply modules/monitoring/doc/03-Configuration.md]: The README still tells it is supported. `supporting Icinga 2, Icinga Core and any other monitoring backend compatible with the IDO database`

Comment (nilmerg) [reply modules/monitoring/doc/03-Configuration.md]: Yes, emphasis on *still*. Though, we shouldn't cover such in newly introduced documentation. It's going to be removed anyway in the future.

**Comment by dgoetz** [comment]
Updated as requested

**Pull request merged**